README.md

```diff
@@ -1,65 +1,256 @@
 # MinIO Operator [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
 
-MinIO Operator brings native support for [MinIO](https://github.com/minio/minio), [Graphical Console](https://github.com/minio/console), and [Encryption](https://github.com/minio/kes) to Kubernetes. This document explains how to get started with MinIO Operator using `kubectl minio` plugin.
+<a href="https://min.io"> <img src=https://raw.githubusercontent.com/minio/minio/master/.github/logo.svg width="600px"/> </a>
 
-## Prerequisites
+MinIO provides Kubernetes-native high performance object storage for private and
+public cloud infrastructure (Hybrid Cloud). The MinIO API maintains compatibility
+with the S3-API to simplify migration of applications onto MinIO Object Storage.
 
-- Kubernetes >= v1.17.0.
-- Create PVs.
-- Install `kubectl minio` plugin using `kubectl krew install minio`.
+## Table of Contents
 
-## Operator Setup
+* [Architecture](https://github.com/minio/operator#Architecture)
+* [Create a MinIO Tenant](https://github.com/minio/operator#Create-a-MinIO-Tenant)
+* [Expand a MinIO Tenant](https://github.com/minio/operator#Expand-a-MinIO-Tenant)
+* [Kubernetes Cluster Configuration](https://github.com/minio/operator#Kubernetes-Cluster-Configuration)
 
-MinIO Operator offers MinIO Tenant creation, management, upgrade, zone addition and more. Operator is meant to control and manage multiple MinIO Tenants.
+# Architecture
 
-To get started, initialize the MinIO Operator deployment. This is a _one time_ process.
+Each MinIO Tenant represents an independent MinIO Object Store service within
+the Kubernetes cluster. The following diagram describes the architecture of as
+MinIO Tenant deployed into Kubernetes:
+
+IMAGE
+
+MinIO provides multiple methods for accessing and managing the MinIO Tenant:
+
+## MinIO Console
+
+The MinIO Console provides a graphical user interface (GUI) for interacting with
+MinIO Tenants. 
+
+IMAGE
```

Comment:
ToDo: Update section for when Console for Operator (Tenant Management) is ready to include something like:

"Kubernetes Administrators can create and manage MinIO Tenants entirely through the Console and without relying on command line interactions." or something.